[2025-03-14T15:15:05.861+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-14T15:15:05.885+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_prep_transformation_model_building.data_validation manual__2025-03-14T15:14:44.276989+00:00 [queued]>
[2025-03-14T15:15:05.901+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_prep_transformation_model_building.data_validation manual__2025-03-14T15:14:44.276989+00:00 [queued]>
[2025-03-14T15:15:05.901+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-03-14T15:15:05.948+0000] {taskinstance.py:2890} INFO - Executing <Task(_PythonDecoratedOperator): data_validation> on 2025-03-14 15:14:44.276989+00:00
[2025-03-14T15:15:06.003+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'data_prep_transformation_model_building', 'data_validation', 'manual__2025-03-14T15:14:44.276989+00:00', '--job-id', '249', '--raw', '--subdir', 'DAGS_FOLDER/data_prep_transformation_model_building.py', '--cfg-path', '/tmp/tmpp_jybmpu']
[2025-03-14T15:15:06.010+0000] {logging_mixin.py:190} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=1245) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-03-14T15:15:06.012+0000] {standard_task_runner.py:105} INFO - Job 249: Subtask data_validation
[2025-03-14T15:15:06.013+0000] {standard_task_runner.py:72} INFO - Started process 1263 to run task
[2025-03-14T15:15:06.139+0000] {task_command.py:467} INFO - Running <TaskInstance: data_prep_transformation_model_building.data_validation manual__2025-03-14T15:14:44.276989+00:00 [running]> on host 0bbddc3bb474
[2025-03-14T15:15:06.337+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='data_prep_transformation_model_building' AIRFLOW_CTX_TASK_ID='data_validation' AIRFLOW_CTX_EXECUTION_DATE='2025-03-14T15:14:44.276989+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-14T15:14:44.276989+00:00'
[2025-03-14T15:15:06.342+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-14T15:15:06.548+0000] {logging_mixin.py:190} INFO - Data Validation Completed. Report saved at:
[2025-03-14T15:15:06.550+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-03-14T15:15:06.578+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-14T15:15:06.579+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=data_prep_transformation_model_building, task_id=data_validation, run_id=manual__2025-03-14T15:14:44.276989+00:00, execution_date=20250314T151444, start_date=20250314T151505, end_date=20250314T151506
[2025-03-14T15:15:06.793+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-14T15:15:06.888+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-03-14T15:15:06.909+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
